src/gui/guiFormSpecMenu.cpp

```diff
@@ -4117,12 +4162,14 @@ bool GUIFormSpecMenu::OnEvent(const SEvent& event)
 	}
 
 	/* Mouse event other than movement, or crossing the border of inventory
-	  field while holding right mouse button
+	  field while holding left, right, or middle mouse button
 	 */
 	if (event.EventType == EET_MOUSE_INPUT_EVENT &&
 			(event.MouseInput.Event != EMIE_MOUSE_MOVED ||
 			 (event.MouseInput.Event == EMIE_MOUSE_MOVED &&
-			  event.MouseInput.isRightPressed() &&
+				 (event.MouseInput.isLeftPressed()
+			   || event.MouseInput.isRightPressed()
+			   || event.MouseInput.isMiddlePressed()) &&
 			  getItemAtPos(m_pointer).i != getItemAtPos(m_old_pointer).i))) {
```

Comment:
Yeah, quite ugly that one, but I don't really think it can be changed much. I did notice a redundant part I could remove though:
```cpp
	if (event.EventType == EET_MOUSE_INPUT_EVENT &&
		(event.MouseInput.Event != EMIE_MOUSE_MOVED ||
			((event.MouseInput.isLeftPressed() ||
				event.MouseInput.isRightPressed() ||
				event.MouseInput.isMiddlePressed()) &&
			getItemAtPos(m_pointer).i != getItemAtPos(m_old_pointer).i))) {
```